<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerForm</class>
 <widget class="QWidget" name="FileManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileManager</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="1" column="0">
      <widget class="QTreeView" name="treeView"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="groupBox">
         <item>
          <property name="text">
           <string>Group by folders</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Group by types</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="displayBox">
         <item>
          <property name="text">
           <string>Display in table</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
